### ISSUE TITLE
IOError: [Errno 2] on dumpgenerator with image filename containing /

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fsrrt/wikiteam#86]
```
I am using dumpgenerator revision r882 on windows 7 and I am getting the 
following error:
Traceback (most recent call last):
  File "dumpgenerator.py", line 1220, in <module>
    main()
  File "dumpgenerator.py", line 1211, in main
    resumePreviousDump(config=config, other=other)
  File "dumpgenerator.py", line 1127, in resumePreviousDump
    generateImageDump(config=config, other=other, images=images, start=lastfilename2) # we resume from previous image, which may be corrupted (or missing .desc)
  by the previous session ctrl-c or abort
  File "dumpgenerator.py", line 673, in generateImageDump
    urllib.urlretrieve(url=url, filename='%s/%s' % (imagepath, filename2) )
  File "C:\Python27\lib\urllib.py", line 94, in urlretrieve
    return _urlopener.retrieve(url, filename, reporthook, data)
  File "C:\Python27\lib\urllib.py", line 244, in retrieve
    tfp = open(filename, 'wb')
IOError: [Errno 2] No such file or directory: 
'./norunescapewikiacom-20140127-wikidump/images/Engelsk/Norsk.png'
```

Original issue reported on code.google.com by `joehow...@gmail.com` on 27 Jan 2014 at 8:09
- Blocked on: #73

Attachments:
- [errno2.png](https://storage.googleapis.com/google-code-attachments/wikiteam/issue-86/comment-0/errno2.png)

**Comment by GoogleCodeExporter** [comment]
```
Another Wikia wiki... I wonder if this is the same as issue 81. Ah, no, it's 
probably that slash in the filename. Does someone know how to handle Windows?
```

Original comment by `nemow...@gmail.com` on 31 Jan 2014 at 3:01
- Changed title: **IOError: [Errno 2] on dumpgenerator with image filename containing /**
- Added labels: **OpSys-Windows**

**Comment by GoogleCodeExporter** [comment]
```
Issue 63 has been merged into this issue.
```

Original comment by `nemow...@gmail.com` on 31 Jan 2014 at 3:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `nemow...@gmail.com` on 31 Jan 2014 at 3:28
- Now blocked on: #73
